### PR TITLE
[sunder] Add sunder strategies

### PR DESCRIPTION
**Pull request by krotos-arch**
Fixes # .

Changes proposed in this pull request:

Sunder is the customized strategy for Sunder protocol. Any governance token been sundered with dToken can choose this strategy to get it included with the original token with   1:1 voting power.

### CODE REVIEW

**Review by krotos-arch — COMMENTED** [review]
Update example json

File: src/strategies/sunder/index.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import Multicaller from '../../utils/multicaller';
+
+export const author = 'aaron';
```
Comment (ChaituVR) [inline src/strategies/sunder/index.ts:5]:
```suggestion
export const author = 'krotos-arch';
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Review by ChaituVR — CHANGES_REQUESTED** [review]
@krotos-arch There is some error:
```
  ● 
Test strategy "sunder" › Strategy should run without any errors

    call revert exception (method="aggregate((address,bytes)[])", errorArgs=null, errorName=null, errorSignature=null, reason=null, code=CALL_EXCEPTION, version=abi/5.3.1)

      at Logger.Object.<anonymous>.Logger.makeError (node_modules/@ethersproject/contracts/node_modules/@ethersproject/abi/node_modules/@ethersproject/logger/src.ts/index.ts:213:28)
      at Logger.Object.<anonymous>.Logger.throwError (node_modules/@ethersproject/contracts/node_modules/@ethersproject/abi/node_modules/@ethersproject/logger/src.ts/index.ts:225:20)
      at Interface.Object.<anonymous>.Interface.decodeFunctionResult (node_modules/@ethersproject/contracts/node_modules/@ethersproject/abi/src.ts/interface.ts:384:23)
      at Contract.<anonymous> (node_modules/@ethersproject/contracts/src.ts/index.ts:291:44)
      at step (node_modules/@ethersproject/contracts/lib/index.js:46:23)
      at Object.next (node_modules/@ethersproject/contracts/lib/index.js:27:53)
      at fulfilled (node_modules/@ethersproject/contracts/lib/index.js:18:58)
```

**Comment by ChaituVR** [comment]
@krotos-arch Hey, I think the error is because your fork branch is outdated from the master branch (many big changes pushed recently), can you please update your branch with those, that should fix this weird issue :)

**Pull request closed without merging**